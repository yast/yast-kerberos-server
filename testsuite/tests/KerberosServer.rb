# encoding: utf-8

module Yast
  class KerberosServerClient < Client
    def main
      # testedfiles: KerberosServer.ycp

      Yast.include self, "testsuite.rb"
      TESTSUITE_INIT([], nil)

      Yast.import "KerberosServer"

      DUMP("KerberosServer::Modified")
      TEST(lambda { KerberosServer.Modified }, [], nil)

      nil
    end
  end
end

Yast::KerberosServerClient.new.main
