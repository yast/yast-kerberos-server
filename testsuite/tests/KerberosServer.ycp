{

// testedfiles: KerberosServer.ycp

include "testsuite.ycp";
TESTSUITE_INIT([], nil);

import "KerberosServer";

DUMP("KerberosServer::Modified");
TEST(``(KerberosServer::Modified()), [], nil);

}
