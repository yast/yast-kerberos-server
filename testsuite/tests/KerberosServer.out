Dump	KerberosServer::Modified
Return	false
